Cue/app/common/DeckThumbnail.js

```diff
@@ -0,0 +1,84 @@
+// @flow
+
+'use strict';
+
+import React from 'react'
+import { View, Text, TouchableOpacity, Dimensions, Navigator } from 'react-native'
+
+import type { Deck } from '../api/types'
+
+import CueColors from './CueColors'
+
+export default class DeckThumbnail extends React.Component {
+  props: {
+    deck: Deck,
+    onPress?: () => void,
+  }
+
+  render() {
+
+    const styles = {
+      itemContainer: {
+        alignItems: 'flex-start',
+        justifyContent: 'center',
+        padding: 4,
+        width: Math.floor((Dimensions.get('window').width - 64) / 2),
+        aspectRatio: 1.35,
+        borderWidth: 2,
+        borderColor: CueColors.primaryText,
+      },
+      contentContainer: {
+        flex: 1,
+        alignItems: 'flex-start',
+        justifyContent: 'center',
+      },
+      itemText: {
+        color: CueColors.primaryText,
+        fontSize: 20,
+      },
+      cardNumber: {
+        fontSize: 12,
+        color: CueColors.lightText,
+        position: 'absolute',
+        bottom: 4,

```

Comment:
Looks like `position: absolute` elements are laid out relative to the padding on the container, so the card number is actually placed as if it had `bottom: 8, left: 8` instead. Please update this to `bottom: 0, left: 0` instead.